Remotion/Web/Development.WebTesting.IntegrationTests/TreeViewControlObjectTest.cs

```diff
@@ -379,8 +381,14 @@ public void TestTreeSelectNodeInHierarchyOnlyRootNodeExpanded ()
 
       treeView.Scope.ElementFinder.Options.Timeout = TimeSpan.Zero;
 
-      Assert.That (() => treeView.GetNodeInHierarchy().WithDisplayText ("Child node 12").Select(), Throws.InstanceOf<ElementNotInteractableException>());
-      Assert.That (() => treeView.GetNodeInHierarchy().WithDisplayTextContains ("11").Select(), Throws.InstanceOf<ElementNotInteractableException>());
+      Assert.That (
+          () => treeView.GetNodeInHierarchy().WithDisplayText ("Child node 12").Select(),
+          Throws.InstanceOf<WebTestException>()
+              .With.Message.EqualTo ("The element cannot be interacted with."));
+      Assert.That (
+          () => treeView.GetNodeInHierarchy().WithDisplayTextContains ("11").Select(),
+          Throws.InstanceOf<WebTestException>()
+              .With.Message.EqualTo ("The element cannot be interacted with."));

```

Comment:
All these exceptions represent the same scenario: "I tried to interact with a node without interacting with its parent first". Shouldn't they also have an exception message that says something along those lines?

Reply:
True, I'll leave that open for the lead review.

Reply:
see prod code comment